lib/ultralib/include/PR/gbi.h

```diff
@@ -1353,11 +1353,11 @@ typedef struct {
 
 typedef struct {
     unsigned char col[3];
-    unsigned char unk3;
+    unsigned char kc;
     unsigned char colc[3];
-    unsigned char unk7;
+    unsigned char kl;
     short pos[3];
-    unsigned char unkE;
+    unsigned char kq;
```

Comment:
What does those names mean?

Reply:
No idea, it's what they're named in GC: https://github.com/Prakxo/ac-decomp/blob/master/include/libforest/gbi_extensions.h#L335-L342

Reply:
The names are exposed in the graphics emulator